cli/src/declarations.ts

```diff
@@ -1,22 +1,340 @@
+    appendUserAgent?: string;
+
+    /**
+     * Background color of the Capacitor Web View for iOS.
+     *
+     * Overrides global `backgroundColor` option.
+     *
+     * @since 1.0.0
+     */
+    backgroundColor?: string;
+
+    /**
+     * Configure the scroll view's content inset adjustment behavior.
+     *
+     * This will set the
+     * [`contentInsetAdjustmentBehavior`](https://developer.apple.com/documentation/uikit/uiscrollview/2902261-contentinsetadjustmentbehavior)
+     * property on the Web View's
+     * [`UIScrollView`](https://developer.apple.com/documentation/uikit/uiscrollview).
+     *
+     * @since 1.0.0
+     * @default never
+     */
+    contentInset?: 'automatic' | 'scrollableAxes' | 'never' | 'always';
+
+    /**
+     * Configure the Swift version to be used in Cordova plugins.
+     *
+     * @since 1.0.0
+     * @default 5.1
+     */
     cordovaSwiftVersion?: string;
+
+    /**
+     * Configure the minimum iOS version supported.
+     *
+     * @since 1.0.0
+     * @default 11.0
+     */
     minVersion?: string;
+
+    /**
+     * Configure custom linker flags for compiling Cordova plugins.
+     *
+     * @since 1.0.0
+     * @default []
+     */
     cordovaLinkerFlags?: string[];
+
+    /**
+     * Allow destination previews when pressing on links.
+     *
+     * @since 1.0.0
+     * @default false
+     */
+    allowsLinkPreview?: boolean;
+
+    /**
+     * Hide or show the native logs for iOS.
+     *
+     * Overrides global `hideLogs` option.
+     *
+     * @since 1.0.0
+     * @default false
+     */
+    hideLogs?: boolean;
   };
+
+  server?: {
+    /**
+     * Configure the local hostname of the device.
+     *
+     * It is recommended to keep this as `localhost` as it allows the use of
+     * Web APIs that would otherwise require a [secure
+     * context](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts)
+     * such as
+     * [`navigator.geolocation`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/geolocation)
+     * and
+     * [`MediaDevices.getUserMedia`](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia).
+     *
+     * @since 1.0.0
+     * @default localhost
+     */
+    hostname?: string;
+
+    /**
+     * Configure the local scheme on iOS.
+     *
+     * This can be useful when migrating from
+     * [`cordova-plugin-ionic-webview`](https://github.com/ionic-team/cordova-plugin-ionic-webview),
+     * where the default scheme on iOS is `ionic`.
+     *
+     * @since 1.0.0
+     * @default capacitor
+     */
+    iosScheme?: string;
+
+    /**
+     * Configure the local scheme on Android.
+     *
+     * @since 1.0.0
+     * @default http
+     */
+    androidScheme?: string;
+
+    /**
+     * Load an external URL in the Web View.
+     *
+     * This is intended for use with live-reload servers.
+     *
+     * **This is not intended for use in production.**
+     *
+     * @since 1.0.0
+     */
+    url?: string;
```

Comment:
Would it be possible to clarify why we say this `This is not intended for use in production`?
You can see under https://github.com/ionic-team/capacitor/discussions/5075 this question was asked. In https://github.com/ionic-team/capacitor/issues/4122 people still use it and it fixes problems like https://github.com/ionic-team/capacitor/issues/1373

Reply:
@imhoffd